docs/sql-programming-guide.md

```diff
@@ -1096,13 +1096,17 @@ the Data Sources API. The following options are supported:
 
 {% highlight sql %}
 
-CREATE TEMPORARY VIEW jdbcTable
+CREATE TEMPORARY TABLE jdbcTable
```

Comment:
Please change it back. `CREATE TEMPORARY TABLE` is deprecated. You will get a Parser error
```
CREATE TEMPORARY TABLE is not supported yet. Please use CREATE TEMPORARY VIEW as an alternative.(line 1, pos 0)
```